### ISSUE TITLE
Chore: investigate mixed concrete supplier/delivery in Braga

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#242]

**Comment by nelsonic** [comment]
@iteles this is the issue you will need today.
Please head to SegSoc and try to revive your conversation about reserving the parking spaces for "ASAP".
The sooner we can get this first concrete pour done the better.

Then while you are in the Q waiting, call Paulo from **`Betão Liz`** to 'splain who you are and that he came to our site.
Tell Paulo: 

## "A cofragem fica pronta hoje para receber a primeira leva de betão, <br />quando podem entregar? Precisamos da bomba para fazer chegar o betão ao sitio e estou a falar com a Segurança Social para reservar os lugares de estacionamento; preciso de saber a vossa disponibilidade para finalizar a reserva."

Obvs it's not ready yet. But just see what he says.
If they can deliver it next week then we can work back from then.
Remember the faster we can move on this the better. 💰 🔥 

If he says he needs to come to the site to inspect, axsk him how soon he can be here. ⏳ 
Thanks. 🙏

**Comment by nelsonic** [comment]
Just measured the South and East walls for calculating concrete order:

<img width="923" alt="image" src="https://github.com/dwyl/home/assets/194400/6f8454d6-4bbb-4c8b-8d91-48e24a38b792">

Plugging these numbers into the concrete calculator: 



https://www.calculator.net/concrete-calculator.html?slablength=17.6&slablengthunit=meter&slabwidth=.3&slabwidthunit=meter&slabthick=2.6&slabthickunit=meter&slabquantity=1&slabcal=Calculate

<img width="760" alt="image" src="https://github.com/dwyl/home/assets/194400/e4672837-30f9-4d7a-a5d9-7fe1fa40b360">

Please send this line to Paulo: 

17.6m (lineares) x 2.7m (altura) x 0.3m (largura) = **13.73 m**<sup>2</sup>
Axsk him if they will send an engineer out to confirm prior to the mix+pour. Thanks.

**Comment by iteles** [comment]
Agreed with Paulo from Betão Liz that we would be getting Betão 25/30 and it would be delivered Tuesday 08h00/08h15 ish.

Costs:
+ Betão is €88.40 + IVA per cubic metre (we agreed 18 cubic metres as each truck holds 9)
+ Bomba is €11 + IVA per cubic metre + €50 for transport

TOTALS: €1,839.20 + IVA
**TOTAL incl IVA**: **€2261.97** for the first pour
